web/src/beta/features/Editor/DataSourceManager/DelimitedText/index.tsx

```diff
@@ -19,10 +21,21 @@ import {
 const DelimitedText: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
   const t = useT();
 
-  const [sourceType, setSourceType] = React.useState("url"); // ["url", "local", "value"]
+  const [sourceType, setSourceType] = React.useState<SourceType>("local");
   const [value, setValue] = React.useState("");
   const [lat, setLat] = React.useState("");
   const [long, setLong] = React.useState("");
+  const [fileFormat, setFileFormat] = React.useState<FileFormatType>("CSV");
+
+  const DataSourceOptions = useMemo(
+    () => [
+      { label: t("From Assets"), keyValue: "local" },
+      { label: t("From Web"), keyValue: "url" },
+    ],
+    [t],
+  );
+
+  const FileFormatOptions = useMemo(() => ["CSV"], []);
```

Comment:
Do we need this? Or are the FileFormatOptions the same as the FileFormatType?
If we still need this, please move it above, outside of the DelimitedText component declaration 🙏🏻